_datascience/proof.md

```diff
@@ -0,0 +1,372 @@
+---
+title: PROOF
+main_authors: vortexing, sitapriyamoorthi
+primary_reviewers: vortexing
+---
+
+## What is PROOF?
+
+**PROOF** (PRoving cOmpute On cloud Foundations) is a user-friendly tool designed for managing and executing [**WDL**](https://docs.openwdl.org/en/1.0.0/) (Workflow Description Language) workflows using the [**Cromwell**](https://cromwell.readthedocs.io/en/stable/) workflow manager, configured to run on the [**Fred Hutch cluster**.](https://sciwiki.fredhutch.org/scicomputing/compute_jobs/) 
+
+PROOF :
+
+- Automates all the backend configurations necessary to run your workflows instantly.
+
+- Allows researchers to validate, troubleshoot, assess performance, and run their workflows all under one roof.
+
+- Also functions as a 'proofing' resource, enabling researchers to refine their workflows before potential transitions to cloud-based infrastructures.
+
+This guide is intended to describe how you can run PROOF, catering to varying levels of computational expertise. 
+
+We also have a WDL 101 course that will be a quick-start guide to building a WDL workflow.
+
+## What is a WDL?
```

Comment:
I think this section is good, but maybe some of the bullet points should be combined to make the list shorter.